Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  Calculator - CalculatorViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+class CalculatorViewController: UIViewController {
+    private var formula: String = ""
+    private let numberFormatter = NumberFormatter()
```

Comment:
넘버포메터는 한번 세팅되고 그 값이 변화하지 않죠? 그렇다면 다음과 같이 선언해주는것도 좋은 방법일 수 있습니다.
실제로, 나중에 코드를 통해 뷰 컴포넌트들을 구현할 시 다음과 같은 방법을 많이 활용하곤 합니다. 참고해보세요~
```suggestion
    private let numberFormatter: NumberFormatter = {
        let formatter = NumberFormatter()
        formatter.numberStyle = .decimal
        formatter.usesSignificantDigits = true
        formatter.maximumFractionDigits = -2
        formatter.maximumIntegerDigits = 20
        formatter.maximumSignificantDigits = 20
        return formatter
    }()
```

Reply:
오! 좋은 방법인 것 같아요! 좋은 조언 감사합니다!!